modules/network/vpn-gateway/vpn-connection/main.bicep

```diff
@@ -54,6 +54,9 @@ param sharedKey string = ''
 @description('Optional. Reference to a VPN site to link to.')
 param remoteVpnSiteResourceId string = ''
 
+@description('Optional. Dead peer detection (dpd) timeout in seconds vpn connection.')
+param dpdTimeoutSeconds int = 45
```

Comment:
@MariusStorhaug when deploying the following error is returned when this value is introduced

```
       - Cannot parse the request. (Code: InvalidRequestFormat)
         - Could not find member 'dpdTimeoutSeconds' on object of type 'VpnConnectionProperties'. Path 'properties.dpdTimeoutSeconds'

```

Reference link ==> https://github.com/Azure/ResourceModules/actions/runs/5994186773/job/16255985999#step:4:718

Have you used this property before?

Reply:
Couldn't get around this error. I'd suggest we take it out and and can be added as a separate PR if requested as it's the only failing networking pipeline that's blocking this PR. Thoughts @MariusStorhaug @AlexanderSehr

Reply:
@AlexanderSehr Should we just remove it for now?

Reply:
Sounds good to me. I don't think we have many options.

Reply:
🚀🚀🚀

Reply:
Re-triggered the pipeline